bundles/org.openhab.binding.bigassfan/src/main/java/org/openhab/binding/bigassfan/internal/handler/BigAssFanHandler.java

```diff
@@ -96,18 +99,19 @@ public BigAssFanHandler(Thing thing, String ipv4Address) {
     public void initialize() {
         logger.debug("BigAssFanHandler for {} is initializing", thing.getUID());
 
-        config = getConfig().as(BigAssFanConfig.class);
-        logger.debug("BigAssFanHandler config for {} is {}", thing.getUID(), config);
+        BigAssFanConfig configuration = getConfig().as(BigAssFanConfig.class);
+        logger.debug("BigAssFanHandler config for {} is {}", thing.getUID(), configuration);
 
-        if (!config.isValid()) {
+        if (!configuration.isValid()) {
```

Comment:
What was the point of changing the name to configuration? Was there something wrong with config?

Reply:
First config was removed as it was class scope and not needed at class level. Then i introduced a local scoped var and gave it this name. Nothing intentionally, nothing wrong with config :-),  just something that happens in the workflow. But if you want config back, i'll make a commit ;-/

Reply:
No, just wanted to understand why it was changed.